website/static/js/contribManager.js

```diff
@@ -462,11 +423,39 @@ var ContributorsViewModel = function(contributors, adminContributors, user, isRe
 
 function ContribManager(selector, contributors, adminContributors, user, isRegistration, table, adminTable) {
     var self = this;
+    //shouter allows communication between ContribManager and ContribRemover, in particular which contributor needs to
+    // be removed is passed to ContribRemover
+    var contribShouter = new ko.subscribable();
```

Comment:
These "shouters" seem unnecessarily complicated. Couldn't you just pass the relevant observables to the `ContribRemover`?

```js
var ContribsVM = function() {
    self.contribToRemove = ko.obs();
    self.changed = ko.obs();
};


var ContribRemover = function(..., contribToRemoveObservable, changedObservable) {
    self.contribtoRemove = contribToRemoveObservable;

    // ...
    self.contribtoRemove(); // will be up-to-date with ContribManager
}

var ContribManager = function() {
    ...

    new ContribRemover(..., self.viewModel.contribToRemove, self.viewModel.changed);
}

```